### ISSUE TITLE
The size of the gama.dependencies plugin is big

### ISSUE THREAD

**Issue by lesquoyb** [gama-platform/gama#110]
### Discussed in https://github.com/gama-platform/gama/discussions/78

<div type='discussions-op-text'>

<sup>Originally posted by **hqnghi88** July  1, 2022</sup>
If we take a look in the list of plugins artifact, easily to see that the msi.gama.ext is the biggest. It is fine if we download whole GAMA package with high speed internet connection. However in some cases, it is hard to make it work well with low-speed network, or in specific configuration, such as the update mechanism of eclipse (still not work well as sometime it fail to download big artifact), compiling/installing/deploying somewhere somehow with the limitation of files (10mb or 50mb only)...
As we know have increase the modularity of gama, and someday with the auto-installation of required packages, i wonder if we can split that big ext into smaller pieces. I tried to split by 2, mostly there is geotools as the big part and the rest. It seem to have no technical problem.
What do you think?</div>

**Comment by lesquoyb** [comment]
@hqnghi88 can you summarize a bit what the issue is and if the solution you initially proposed is still possible to do with the new architecture ?

**Comment by hqnghi88** [comment]
Hi, I cant test the new structure as the local build script seems not working, neither i did not find the new binary on github release. I will return later on this.
